tools/cpp/osx_cc_configure.bzl

```diff
@@ -64,7 +64,7 @@ def configure_osx_toolchain(repository_ctx, overriden_tools):
     ])
 
     xcode_toolchains = []
-    (xcode_toolchains, xcodeloc_err) = run_xcode_locator(
```

Comment:
Why the removal here? even in cases where xcode isn't on the location machine, a comment (in crosstool or build files) ends up sometimes providing us a nice debugging signal

Reply:
It still ends up in the BUILD file from here: https://github.com/bazelbuild/bazel/pull/7519/files#diff-91c75e949de96040772e8ca69cff0e2cR171